tooling/v8-snapshot/cache/dev-darwin/snapshot-meta.cache.json

```diff
@@ -632,6 +621,15 @@
     "./node_modules/xml2js/lib/xml2js.js",
     "./node_modules/yauzl/index.js",
     "./node_modules/zip-stream/index.js",
+    "./packages/config/node_modules/@babel/traverse/lib/context.js",
```

Comment:
I see these are not deterministic - at least, the _order_ isn't. I wonder if we can sort them out. Let's ask @ryanthemanuel - this might help with not cluttering up diffs in the future.

Reply:
These should be alphabetical?

Reply:
Oh, I see. I wonder why these changed here, then? This PR does not touch and dependencies. Or, it is left over from a previous PR?

Reply:
Probably a previous one.